website/docs/developer/deployment-guide/deploying-from-windows.mdx

```diff
@@ -0,0 +1,185 @@
+      <img alt="Location of Account Id" src={useBaseUrl('img/aws-account-id.PNG')} />
+    - Set ```rds_user_access_secret_arn``` to the ```user_credentials_secret_urn``` output from ```terraform apply```.
+    - Set ```rds_security_group``` to the ```security_group_id``` output from ```terraform apply```.
+    - Set ```vpc_id``` to your borrowed VPC.
+
+```bash
+cd ../data-persistence-tf/
+aws configure
+terraform init
+terraform plan
+terraform apply
+```
+
+- In cumulus-tf/terraform.tfvars
+    - Add the line ```aws_profile = "default"```
+    - Replace 12345 in permissions_boundary_arn with the Account Id.
+    - Add to the buckets:
+      ```
+      default_orca = {
+         name = "prefix-orca"
+         type = "orca"
+        },
+        provider = {
+          name = "orca-sandbox-s3-provider"
+          type = "provider"
+      }
+      ```
+      :::tip
+      The "orca-sandbox-s3-provider" bucket contains test data.
+      If creating a separate environment, you can create your own bucket.
+      :::
+
+    - Replace the ```cumulus_message_adapter_lambda_layer_version_arn``` with a valid one.
+
+      :::warning
+      Process for finding the correct arn is not documented.
+      :::
+-
+    - Set the ```ecs_cluster_instance_image_id``` to ```""```
+    - ```ecs_cluster_instance_subnet_ids``` and ```lambda_subnet_ids``` should have the same two values.
+    - Set ```urs_client_id``` and ```urs_client_password``` to the values from your created application.
+    - Add an extra property ```urs_url = "https://uat.urs.earthdata.nasa.gov"```
+    - Add your username to the ```api_users```
+    - Set ```token_secret``` to a value of your choice.
+    - Comment out the ```archive_api_port``` property and value.
+    - Uncomment the ```key_name property``` and set the value to ```"PREFIX-key-pair"```
+    - Add this section to the bottom of the file and edit as desired:
+      ```
+      ## =============================================================================
+      ## ORCA Variables
+      ## =============================================================================
+
+      ## REQUIRED TO BE SET
+      ## -----------------------------------------------------------------------------
+
+      ## ORCA application database user password.
+      database_app_user_pw = "This1sAS3cr3t"
+
+      ## Default ORCA S3 Glacier bucket to use
+      orca_default_bucket = PREFIX-glacier"
+
+      ## PostgreSQL database (root) user password
+      postgres_user_pw = "An0th3rS3cr3t"
+      ```
+    :::warning
+    The instructions in the tfvars file suggest swapping '12345' with your account ID. This may not work, depending on how your dependencies such as the cumulus_message_adapter_lambda_layer_version_arn were set up.
+    :::
+- In cumulus-tf/orca.tf:
+    - You may change ```source``` to an alternate release. If local, make sure it is within the scope of the container.
+```bash
```

Comment:
May want to space this under the bullet point

Reply:
Added